src/views/Files.vue

```diff
@@ -52,7 +52,7 @@ export default {
     Preview
   },
   computed: {
-    ...mapGetters(["selectedCount", "isListing", "isEditor", "isFiles", "isActiveDialog", "shares", "direction", "isSearch"]),
+    ...mapGetters(["isPreivew", "selectedCount", "isListing", "isEditor", "isFiles", "isActiveDialog", "shares", "direction", "isSearch"]),
```

Comment:
I think you misspelled `preview `

Reply:
Check where else it might have happened and update the Kubernetes

Reply:
it wasn't used so i deleted that